### PR TITLE
ROS2 complian table

### PR DESCRIPTION
**Pull request by julianbermudez**
This pull request adds a ROS2 compliant table in the Overview section.
This tabla contains links to installation documentation and `.repos` files.

### CODE REVIEW

File: _docs/overview/index.md [hunk 1]
```diff
@@ -34,3 +34,15 @@ Major repositories in order of the layers are:
   * Example applications for NuttX directly are in [apps](https://github.com/micro-ROS/apps)
 
 Most repositories can be found in GitHub's micro-ROS organization at [github.com/micro-ROS/](https://github.com/micro-ROS/).
+
+### ROS2 Compliant Table
+
+The following table summarizes the compatibility between micro-ROS and ROS2 releases.
```
Comment (ralph-lange) [inline _docs/overview/index.md]:
```suggestion
The following table summarizes the compatibility between micro-ROS and ROS 2 releases.
```
Open Robotics typically writes ROS 2 with a space in-between.

File: _docs/overview/index.md [hunk 2]
```diff
@@ -34,3 +34,15 @@ Major repositories in order of the layers are:
...
+On the other hand, the *Agent* size `.repos` includes the micro-ROS-Agent bridge.
+
+| ROS2 Release | Documentation | Client | Agent  |
+|:-------------|:--------------|:-------|:-------|
+| **Crystal**  | [![](https://img.shields.io/badge/read-the%20docs-blue)](https://github.com/micro-ROS/micro-ROS-doc/blob/crystal/Installation) | [![](https://img.shields.io/badge/uROS-repos-brightgreen)](https://github.com/micro-ROS/micro-ROS-doc/blob/crystal/Installation/repos/client_minimum.repos) | [![](https://img.shields.io/badge/uROS-repos-brightgreen)](https://github.com/micro-ROS/micro-ROS-doc/blob/crystal/Installation/repos/agent_minimum.repos) |
+| **Dashing**  | [![](https://img.shields.io/badge/read-the%20docs-blue)](https://github.com/micro-ROS/micro-ROS-doc/blob/dashing/Installation) | [![](https://img.shields.io/badge/uROS-repos-brightgreen)](https://github.com/micro-ROS/micro-ROS-doc/blob/dashing/Installation/repos/client_minimum.repos) | [![](https://img.shields.io/badge/uROS-repos-brightgreen)](https://github.com/micro-ROS/micro-ROS-doc/blob/dashing/Installation/repos/agent_minimum.repos) |
```
[the remark this answers is not in the document]
Comment (ralph-lange) [reply _docs/overview/index.md]: Note that you may use generated badge from the CI job but nevertheless link the repo or .repos file.

File: _docs/overview/index.md [hunk 3]
```diff
@@ -34,3 +34,15 @@ Major repositories in order of the layers are:
   * Example applications for NuttX directly are in [apps](https://github.com/micro-ROS/apps)
 
 Most repositories can be found in GitHub's micro-ROS organization at [github.com/micro-ROS/](https://github.com/micro-ROS/).
+
+### ROS2 Compliant Table
+
+The following table summarizes the compatibility between micro-ROS and ROS2 releases.
+This table contains links to installation documentation as well as `.repos` file for both *Client* and *Agent* side.
+On the one hand, the *Client* size `.repos` includes all the necessary to build a micro-ROS application, along with some demo examples.
```
[the remark this answers is not in the document]
Comment (julianbermudez) [reply _docs/overview/index.md]: Yes, of course. I have not time yet to work with Ingo's build system, but it is in my mind to move to it.

File: _docs/overview/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (julianbermudez) [reply _docs/overview/index.md]: Ok, I will change it with the https://uuvsimulator.github.io/ template. I just though that you want only links to .repos files.

[the remark this answers is not in the document]
Comment (ralph-lange) [reply _docs/overview/index.md]: Hi @julianbermudez , you do not have to use the whole uuvsimulator template. That's not necessary. However, as a reader I'd feel kidded when recognizing that the green badges have been linked directly. Please use the badge for one or few major repositories (i.e. the corresponding CI).

**Pull request merged**